service/src/main/java/org/jboss/aerogear/unifiedpush/service/impl/spring/KeycloakServiceImpl.java

```diff
@@ -397,8 +393,68 @@ public String strip(String fqdn) {
 
 		return StringUtils.EMPTY;
 	}
-	
+
 	public String seperator() {
 		return conf.getRooturlMatcher().seperator();
 	}
+
+    public static final int BULK_SIZE = 100;
+
+    public static final ObjectMapper MAPPER = new ObjectMapper();
+
+    public static final ObjectWriter WRITER = MAPPER.writer();
+
```

Comment:
I am not sure it is a good idea to make such objects `public` - especially since one could conceivable re-configure the mapper in an incompatible way...